docs/technical-reference/VsTest internal.md

```diff
@@ -39,3 +39,12 @@ Note that this also applies to session cancelled by the timeout logic. When this
 session will still wait for the end session event
 - Providing an incorrect TestSettings file will result in nothing happening, without any error report. This happens if the file is corrupted as well as with a negative timeouts
 - Some settings will be reported in the logs as not recognized or obsolete, but they are still needed by some test adapters
+- The error message `process failed to connect to vstest.console` might indicate that the OS is providing a different version of OpenSSL
+  than what VsTest expects. F. ex. Arch provides OpenSSL3, whereas VsTest expects OpenSSL1. A more
```

Comment:
I don't think `F. ex.` is a common abbreviation for for example. Use `e.g.` or spell it out completely instead.